.env.example

```diff
@@ -37,7 +45,7 @@ DB_LOG_SQL=false
 DB_LOG_SQL_EXPLAIN=false #only for MySQL
 
 # List foreign keys in diagnostic page
-DB_LIST_FOREIGN_KEYS=true
+DB_LIST_FOREIGN_KEYS=false
```

Comment:
Probably makes sense, just checking this wasn't committed accidentally

Reply:
Yeah it just reduces noise in the diagnostics.